similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/AbstractPatientSimilarityView.java

```diff
@@ -181,6 +188,28 @@ public String getContactToken()
         return this.contactToken;
     }
 
+    private String getAgeOfOnset(PatientData<List<SolrVocabularyTerm>> globalControllers) {
+        if (globalControllers != null) {
+            List<SolrVocabularyTerm> modeTermList = globalControllers.get("global_age_of_onset");
```

Comment:
You should probably rename `modeTermList` to something related to "age of onset" to make the code more readable.